### PR TITLE
Fixed .setoffhand command

### PR DESCRIPTION
**Pull request by NRGJobro**
:)

### CODE REVIEW

**Comment by NRGJobro** [comment]
This is not troll

**Comment by Thekingpenguins** [comment]
sexy p

File: Horion/Command/Commands/setoffhandCommand.cpp [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #include "setoffhandCommand.h"
+
+#include "../../../SDK/Tag.h"
 #include "../../../Utils/Utils.h"
-#include <string>
-#include <vector>
 
-setoffhandCommand::setoffhandCommand() : IMCCommand("setoffhand", "Puts any item into offhand", "<ItemName> <count> <itemData>") {
+setoffhandCommand::setoffhandCommand() : IMCCommand("setoffhand", "spawn items in ur offhand", "<itemName> <count> <itemData>") {
```
Comment (FloppyDolphin57) [inline Horion/Command/Commands/setoffhandCommand.cpp]: why

Comment (NRGJobro) [reply Horion/Command/Commands/setoffhandCommand.cpp]: Why not

Comment (NRGJobro) [reply Horion/Command/Commands/setoffhandCommand.cpp]: Verify it pls

Comment (FloppyDolphin57) [reply Horion/Command/Commands/setoffhandCommand.cpp]: that does not look as good

**Comment by GodGamer029** [comment]
Can anyone confirm that this does indeed work?

**Comment by FloppyDolphin57** [comment]
> Can anyone confirm that this does indeed work?
it looks like it would

**Comment by Intoprelised** [comment]
> Can anyone confirm that this does indeed work?

I can confirm I tested it and it does work

**Pull request merged**